README.md

```diff
@@ -37,7 +37,7 @@ You can also upload your data into a catalog source
 
 ```js
 async function main() {
-  const source = new CatalogSource('my_api_key', 'my_coveo_organization_id');
+  const source = new CatalogSource('my_api_key', 'my_coveo_organization_id', {maxRetries=10, retryAfter: 2000, timeMultiple: 3});
```

Comment:
I would remove this, and instead do the retry example in the below section.

Reply:
Also `maxRetries=10` is not a valid JSON object

Reply:
Whoops, brain was still set to "python mode" 🤦 Thanks for catching this